src/graphql/data/nft/Asset.ts

```diff
@@ -187,7 +188,7 @@ export function useAssetsQuery(
           smallImageUrl: asset.smallImage?.url,
           tokenId: asset.tokenId,
           tokenType: asset.collection?.nftContracts[0]?.standard,
-          // totalCount?: number, // TODO waiting for BE changes
+          totalCount: data.nftAssets?.totalCount,
```

Comment:
do you need to default this or it will handle undefined/null?

Reply:
The totalCount type is optional on GenieAsset so it is handled where it is implemented.